src/App.tsx

```diff
@@ -1,56 +1,123 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React, { useState, useMemo } from 'react';
 import './App.scss';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = useState('');
+  const [born, setBorn] = useState(peopleFromServer[0].born);
+  const [died, setDied] = useState(peopleFromServer[0].died);
+  const [selectedName, setSelectedName] = useState('');
+  const [selectedPerson, setSelectedPerson] = useState(peopleFromServer[0]);
+  const [isDebouncing, setIsDebouncing] = useState(false);
+  let isEntered = false;
+  let currentPerson = peopleFromServer[0].name;
```

Comment:
`selectedPerson` and `currentPerson` looks very similar, is it possible to combine them? 

Also, pay attention to previous comments and avoid using `let` in this case.